### ISSUE TITLE
Mac Importation not working

### ISSUE THREAD

**Issue by lzammit** [inverse-inc/packetfence#4684]
I have tested on PacketFence 8.3 (Old admin )and Packetfence 9 (New + old admin)

CSV file:

````
a8:66:7f:18:5c:05
a8:66:7f:18:5c:06
````

PacketFence 8.3 (old admin):
````
Jul 23 04:49:47 pfence1 httpd_admin: httpd.admin(7689) ERROR: [mac:unknown] Trying to save a NULL value in a non nullable field node.unregdate (pf::dal::validate_field)
Jul 23 04:49:47 pfence1 httpd_admin: httpd.admin(7689) ERROR: [mac:unknown] Skipping invalid value (NULL) in when inserting field node.unregdate (pf::dal::_insert_data)
Jul 23 04:49:47 pfence1 httpd_admin: httpd.admin(7689) ERROR: [mac:unknown] Trying to save a NULL value in a non nullable field node.unregdate (pf::dal::validate_field)
Jul 23 04:49:47 pfence1 httpd_admin: httpd.admin(7689) ERROR: [mac:unknown] Skipping invalid value (NULL) in when updating field node.unregdate (pf::dal::_update_data)
Jul 23 04:49:47 pfence1 httpd_admin: httpd.admin(7689) INFO: [mac:unknown] violation 1300003 force-closed for a8:66:7f:18:5c:05 (pf::violation::violation_force_close)
Jul 23 04:49:47 pfence1 httpd_admin: httpd.admin(7689) INFO: [mac:unknown] Instantiate profile PF-Open (pf::Connection::ProfileFactory::_from_profile)
Jul 23 04:49:47 pfence1 httpd_admin: httpd.admin(7689) INFO: [mac:unknown] Created node a8:66:7f:18:5c:05 (pfappserver::Model::Node::create)
Jul 23 04:52:21 pfence1 httpd_admin: httpd.admin(7689) INFO: [mac:unknown] Category 'undef' (pf::import::nodes)
Jul 23 04:52:21 pfence1 httpd_admin: httpd.admin(7689) INFO: [mac:unknown] Category 'undef' (pf::import::nodes)
Jul 23 04:52:21 pfence1 httpd_admin: httpd.admin(7689) INFO: [mac:unknown] violation 1300003 force-closed for a8:66:7f:18:5c:06 (pf::violation::violation_force_close)
Jul 23 04:52:21 pfence1 httpd_admin: httpd.admin(7689) INFO: [mac:unknown] Instantiate profile default (pf::Connection::ProfileFactory::_from_profile)
Jul 23 04:52:21 pfence1 httpd_admin: httpd.admin(7689) INFO: [mac:unknown] CSV file (mac-import.csv) import 0 nodes, skip 2 nodes (pf::import::nodes)
````

Looks like we need to set an unregdate, we never put any unregdate before and it worked.

PacketFence 9.0.1 (old admin):

````
Jul 23 05:02:12 pf1 httpd_admin: httpd.admin(3068) INFO: [mac:unknown] Category 'undef' (pf::import::nodes)
Jul 23 05:02:12 pf1 httpd_admin: httpd.admin(3068) ERROR: [mac:unknown] Trying to save a NULL value in a non nullable field node.unregdate (pf::dal::validate_field)
Jul 23 05:02:12 pf1 httpd_admin: httpd.admin(3068) ERROR: [mac:unknown] Skipping invalid value (NULL) in when inserting field node.unregdate (pf::dal::_insert_data)
Jul 23 05:02:12 pf1 httpd_admin: httpd.admin(3068) ERROR: [mac:unknown] Trying to save a NULL value in a non nullable field node.unregdate (pf::dal::validate_field)
Jul 23 05:02:12 pf1 httpd_admin: httpd.admin(3068) ERROR: [mac:unknown] Skipping invalid value (NULL) in when inserting field node.unregdate (pf::dal::_insert_data)
Jul 23 05:02:12 pf1 httpd_admin: httpd.admin(3068) INFO: [mac:unknown] security_event 1300003 force-closed for a8:66:7f:18:5c:05 (pf::security_event::security_event_force_close)
Jul 23 05:02:12 pf1 httpd_admin: httpd.admin(3068) INFO: [mac:unknown] Instantiate profile default (pf::Connection::ProfileFactory::_from_profile)
Jul 23 05:02:12 pf1 httpd_admin: httpd.admin(3068) INFO: [mac:unknown] Category 'undef' (pf::import::nodes)
Jul 23 05:02:12 pf1 httpd_admin: httpd.admin(3068) ERROR: [mac:unknown] Trying to save a NULL value in a non nullable field node.unregdate (pf::dal::validate_field)
Jul 23 05:02:12 pf1 httpd_admin: httpd.admin(3068) ERROR: [mac:unknown] Skipping invalid value (NULL) in when inserting field node.unregdate (pf::dal::_insert_data)
Jul 23 05:02:12 pf1 httpd_admin: httpd.admin(3068) ERROR: [mac:unknown] Trying to save a NULL value in a non nullable field node.unregdate (pf::dal::validate_field)
Jul 23 05:02:12 pf1 httpd_admin: httpd.admin(3068) ERROR: [mac:unknown] Skipping invalid value (NULL) in when updating field node.unregdate (pf::dal::_update_data)
Jul 23 05:02:12 pf1 httpd_admin: httpd.admin(3068) ERROR: [mac:unknown] Trying to save a NULL value in a non nullable field node.unregdate (pf::dal::validate_field)
Jul 23 05:02:12 pf1 httpd_admin: httpd.admin(3068) ERROR: [mac:unknown] Skipping invalid value (NULL) in when inserting field node.unregdate (pf::dal::_insert_data)
Jul 23 05:02:12 pf1 httpd_admin: httpd.admin(3068) INFO: [mac:unknown] CSV file (mac-import.csv) import 0 nodes, skip 2 nodes (pf::import::nodes)
````

PacketFence 9.0.1 (new admin):

![image](https://user-images.githubusercontent.com/5261214/61710564-e5a68e00-ad1f-11e9-8c1f-b86777a640b3.png)

Only import the second Mac address.

**Comment by satkunas** [comment]
@lzammit the first line is assumed to be a header (default).

To include the first line go to CSV Parser Options and disable Header.

**Comment by nqb** [comment]
@julsemaan, we could also put in maintenance/8.3 ?

**Comment by julsemaan** [comment]
I've put the old admin fix in maintenance/8.3

**Comment by julsemaan** [comment]
and the other 8.x maintenances
